api/mobile.py

```diff
@@ -0,0 +1,12 @@
+from flask import jsonify
+
+
+MIN_ANDROID_VERSION = "0.34.1"
+
+
+def register_endpoints(app):
+    @app.route('/v1/mobile/android/update', methods=['GET'])
```

Comment:
This `update` sounds a little strange. Maybe something like `/v1/mobile/android/versions`?

Reply:
O simply `/v1/mobile/android/min_ver` and as response it could give version string, not a json.